Scripts/Mend SCA/README.md

```diff
@@ -277,3 +278,35 @@ chmod +x ./cache-ua.sh.sh && ./cache-ua.sh.sh
 ```
 
 See additional example for implementation within a build pipeline under [CI-CD](../CI-CD/README.md#caching-the-unified-agent) (`*-cached-ua.yml`).  
+
+<br />
+<hr />
+
+# Get all Users that are part of Organizations which are a part of a Global Organization
+
+[get-all-users-under-global-org.py](get-all-users-under-global-org.py)
+
+This script allows a user to retrieve every single user account that is a part of any of their Organizations which reside under the "umbrella" of a Global Organization. This allows an organization to take a full inventory of all users that have access to their resources.
+
+The [get-all-users-under-global-org.py](get-all-users-under-global-org.py) script can be run with Python 3.9+ and utilizes the Mend API 1.4 as well as API 2.0 to get the information necessary.
+
+> **_NOTE:_** If an error occurs with this script that produces an error when attempting to log in via API 2.0, this can happen on a rare occasion. The best thing to do is to implement a more comprehensive retry logic, or just try running the script again.
+
+<br />
+
+**Prerequisites:**  
+
+* `python 3.9+`
+
+**Execution:**
+```shell
+# Create Environment Variables
+export WS_URL="https://saas.whitesourcesoftware.com"
```

Comment:
update as noted below with [MEND_URL and MEND_EMAIL](https://docs.mend.io/bundle/integrations/page/authenticate_your_login_for_the_mend_cli.html#Mend-CLI---mend-login-parameters)